integration/help_test.go

```diff
@@ -25,7 +25,6 @@ import (
 
 func TestHelpCommand(t *testing.T) {
 	out, err, exitcode := ChefAnalyze("help")
-	verifyGlobalFlagsFromOutput(t, out.String())
```

Comment:
Looks like this is no longer referenced - should we be checking it in a different context, or is it safe to delete?

Reply:
Good catch, I will remove it.